Client/src/components/common/undermaintenancepage/undermaintenancepage.jsx

```
@@ -0,0 +1,78 @@
+            <span className="relative flex items-center justify-center">
+              <ChevronLeft className="mr-2" size={18} /> Go Back
+            </span>
+          </button>
+          
+          {/* Home button */}
+          <button 
+            onClick={() => navigate('/')}
+            className="relative group px-6 py-3 bg-blue-600 text-white font-semibold rounded-full overflow-hidden shadow-lg"
+            type="button"
+          >
+            <span className="absolute inset-0 w-full h-full bg-gradient-to-r from-blue-500 to-blue-700 rounded-full shadow-md"></span>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)